scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -53,8 +53,13 @@ abstract class LocalScanner : Scanner() {
         getPathFromEnvironment(scannerExe)?.parentFile?.takeIf {
             getVersion(it) == scannerVersion
         } ?: run {
-            println("Bootstrapping scanner '$this' as version $scannerVersion was not found in PATH.")
-            bootstrap()
+            if (scannerExe.isNotEmpty()) {
+                println("Bootstrapping scanner '$this' as version $scannerVersion was not found in PATH.")
+                bootstrap()
+            } else {
+                println("Skipping to bootstrap scanner '$this' as it has no executable.")
+                File("")
```

Comment:
Should we allow `null` values here? I didn't check but maybe the code using `scannerDir` could better handle this exception then.

Reply:
Approved anyway, maybe the best cleanup would be to separate between `LocalScanners` that use an executable and those that don't by inheritance, but we can do that later.

Reply:
Agreed.